yscope-compose/demo-assets/init.sh

```
@@ -0,0 +1,50 @@
+    sed -i "s|\${REPLACE_IP}|$ip|g" "$file"
+    echo "Replaced \${REPLACE_IP} with $ip in $file"
+}
+
+function update_metadata_config {
+    if [[ $# -ne 1 ]]; then
+        echo "Usage: update_metadata_config </path/to/clp-package>"
+        return 1
+    fi
+ 
+    local clp_pkg_home=$1
+    local clp_config_path="$(readlink -f ${clp_pkg_home})/etc/clp-config.yml"
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)